backend/apis/users/patch-user.sql

```diff
@@ -0,0 +1,12 @@
+UPDATE users SET accessLevel = ? WHERE id = ?;
+
+INSERT IGNORE INTO
+  userPermissions (userId, permission)
+  SELECT ?, 'immigration' WHERE ?
+;
+
+DELETE FROM
+  userPermissions
+  WHERE
+  userId = ? AND permission = 'immigration' AND ?
```

Comment:
The final `AND ?` is a way of doing a conditional delete. We inject `AND false` when we don't want to delete